### PR TITLE
CSS refactor

### LINKED ISSUE #6: Refactor CSS
Refactor CSS so that the site design stays the same but follows an improved structure

- [x] color theming
- [x] typography
- [x] spacing
- [x] layout and grids
- [ ] rework UI components as needed

### PR DESCRIPTION
**Pull request by jayperryworks**
## Description
- set up new CSS files using current boilerplate and config
- CSS is added to `webpack_assets` folders but not implemented/referenced in any templates yet - will incorporate new classes and markup as we make updates to pages in subsequent sprints.
- set up sprites for current icons (spritemap generated/configured but not implemented - will work on this next sprint)

## Issues
- Resolves #6

### CODE REVIEW

File: webpack_config/base.config.js [hunk 1]
```diff
@@ -29,6 +31,8 @@ const commonConfig = {
   }
 }
 
+console.log(`MODE: ${commonConfig.mode}`)
```
Comment (sschafft) [inline webpack_config/base.config.js]: do we want to kill this in prod?

Comment (sschafft) [reply webpack_config/base.config.js]: Though I guess this is just in webpack so nvm

Comment (jayperryworks) [reply webpack_config/base.config.js]: Oh, sorry, forgot to delete that. Thanks for catching.

Comment (jayperryworks) [reply webpack_config/base.config.js]: Also, the readme is updated with local env info from last PR. No major changes to pipeline in this branch.

**Pull request merged**